### PR TITLE
Fix bugs and update news

### PR DESCRIPTION
**Pull request by ng-anhhtuann**
Fix bugs in Post Methods
Add 4 Get Methods (admin rights)

### CODE REVIEW

File: src/main/java/com/example/GHTK/Model/AdminRight/Area.java [hunk 1]
```diff
@@ -15,7 +15,7 @@ public String toString() {
                 "nameArea='" + nameArea + '\'' +
                 '}';
     }
-
+    public Area(){}
```
Comment (uranashel44) [inline src/main/java/com/example/GHTK/Model/AdminRight/Area.java:18]: Formated yet?

File: src/main/java/com/example/GHTK/Model/AdminRight/Service.java [hunk 2]
```diff
@@ -8,13 +8,10 @@
...
 public class Service {
 
-//    @SerializedName("idService")
-//    @Expose
-//    private String idService;
     @SerializedName("nameService")
     @Expose
     private String nameService;
-
+    public Service(){}
```
Comment (uranashel44) [inline src/main/java/com/example/GHTK/Model/AdminRight/Service.java:14]: same above

**Review by uranashel44 — APPROVED** [review]

File: src/main/java/com/example/GHTK/Model/AdminRight/Area.java (hunk 1 above)
Comment (ng-anhhtuann) [reply src/main/java/com/example/GHTK/Model/AdminRight/Area.java:18]: this is necessary in order not to get error 500 in post object in postman

Comment (uranashel44) [reply src/main/java/com/example/GHTK/Model/AdminRight/Area.java:18]: Area(){}, formatted yet this here

Comment (ng-anhhtuann) [reply src/main/java/com/example/GHTK/Model/AdminRight/Area.java:18]: okay mb

**Pull request merged**